packages/hoppscotch-common/src/components/importexport/ImportExportSteps/MyCollectionImport.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <div class="select-wrapper">
+    <select
+      v-model="mySelectedCollectionID"
+      autocomplete="off"
+      class="select"
+      autofocus
+    >
+      <option :key="undefined" :value="undefined" disabled selected>
```

Comment:
Is the `value` set to `undefined` on purpose?

Reply:
i don't think so. old code.